cloudant-http/src/main/java/com/cloudant/http/internal/interceptors/CookieInterceptorBase.java

```diff
@@ -211,6 +222,15 @@ public boolean call(HttpConnection connection) throws IOException {
                         if (sessionHasStarted(connection.responseAsInputStream())) {
                             return storeCookiesFromResponse(connection.getConnection());
                         } else {
+                            // If the session did not start, consume the error stream
+                            InputStream errorStream = connection.getConnection().getErrorStream();
+                            if (errorStream != null) {
```

Comment:
This same block of code is used on this [line](https://github.com/cloudant/java-cloudant/pull/371/files#diff-fe23e1df4307844289ddf96bd367bae8R131) and in [HttpConnection](https://github.com/cloudant/java-cloudant/pull/371/files#diff-03970475f79bf0e33474af285ca0cb6fR363).  Maybe put this in a utils class to reduce duplicate code?

Reply:
Done in 0687f06cc3dee69903fac1e48661151656056a32.